nimiSlides.nim

```diff
@@ -176,57 +182,71 @@ template initReveal*() =
     fragmentCore(endAnimations = @[@[endAnimation]]):
       body
 
-  template animateCode(lines: varargs[HSlice], body: untyped) =
+  template animateCode(lines: varargs[HSlice[int, int]], body: untyped) =
```

Comment:
I just realized that in order to support both int and the int slice, probably you could do as echo does that applies an operator to all (untyped) elements inside varargs (in that case is $, in this case could be a toHSlice[int, int]) (instead of an approach based on a converter)